spec/Section 2 -- Language.md

```diff
@@ -823,7 +823,8 @@ empty strings.
 
 Note: Unicode characters are allowed within String value literals, however
 {SourceCharacter} must not contain some ASCII control characters so escape
-sequences must be used to represent these characters.
+sequences must be used to represent these characters. Also {`\`}, {`"`} and {LineTerminator}
```

Comment:
@andimarek Block strings can contain unescaped `{`"`} and {LineTerminator}`

Reply:
Block string don't allow any escape sequences except `\"""` so this whole sections only applies to non block strings.

Reply:
But it is a bit confusing because it refers to "String value literals` which according to the grammar are actually normal Strings and Block Strings. I guess this was written before Block Strings was a thing.

Reply:
I updated the section to talk about standard quoted string, to make it more clear.

Reply:
@andimarek Thanks for the clarification 👍